sections/header.liquid

```diff
@@ -156,7 +156,7 @@
                 </nav>
                 <div class="menu-drawer__utility-links">
                   {%- if shop.customer_accounts_enabled -%}
-                    <a href="{%- if customer -%}{{ routes.account_url }}{%- else -%}{{ routes.account_login_url }}{%- endif -%}" class="menu-drawer__account link link--text focus-inset h5">
+                    <a href="{%- if customer -%}{{ routes.account_url }}{%- else -%}{{ routes.account_login_url }}{%- endif -%}" class="menu-drawer__account link focus-inset h5">
```

Comment:
Good catch, Ludo! The reason I removed it is because I didn't want `.link--text:hover {color: rgba(var(--color-foreground), 0.75);}` anymore. And I don't think it makes sense to overwrite the hover.... Maybe I can add the color to the icon themselves. I think that makes the most sense.

Updated it here: https://github.com/Shopify/dawn/pull/576/commits/9e0a5b180a59e7668c4ae76bb93faceefc45c56d